lib/posix-process/exportsyms.uk

```diff
@@ -42,4 +42,7 @@ uk_posix_process_create
 uk_posix_process_kill
 clone
 vfork
+kill
 sigaltstack
+tgkill
+tkill
```

Comment:
Does this have a libc wrapper?
From `man tkill`:
```
Note: glibc provides no wrapper for tkill(), necessitating the use of syscall(2).
```

Reply:
Nice catch, thanks 👍🏼